src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+/**
+ *
+ * @param {Array} arr
+ * mutates Array inplace
+ */
```

Comment:
For JSDoc comments, put the description at the top and include an `@return` annotation when something is returned.  Also, remember to put this function in utils.js and import it here as it's a general function that might be used elsewhere.
```suggestion
/**
 * Mutates Array in place
 * @param {Array} arr the Array to shuffle
 * @return {Array} the shuffled Array
 */
```